backend/src/main/java/com/google/sps/utils/RecipeCollector.java

```diff
@@ -68,4 +70,41 @@
     }
     return recipes;
   }
+
+  public static Entity getRecipeEntity(JsonObject recipe) {
```

Comment:
nit: namethis getRecipeEntityForTesting and annotate method with @VisibleForTesting

Reply:
This method is not only used by tests, it is used in HistoryServlet class as well, so I don't think this applies to it.